src/Tickets/Emails/Email_Abstract.php

```diff
@@ -171,7 +173,7 @@ abstract public function get_settings_fields(): array;
 	 *
 	 * @param array $args The arguments.
 	 *
-	 * @return string The email preview context.
+	 * @return array The email preview context.
```

Comment:
```suggestion
	 * @return array<string,mixed> The email preview context.
```
or whatever the format is.

Rinse and repeat below.